common/dev/pcc.c

```diff
@@ -174,11 +173,18 @@ void check_ABL_error(uint32_t postcode)
 	msg->data[9] = 0xFF; // DIMM Channel
 	msg->data[10] = 0xFF; // DIMM Slot
 	msg->data[11] = 0xFF; // Reserved
-	msg->data[12] = 0x80; // DIMM Error Type
 	msg->data[13] = ABL_error_code_list[i].event_data; // Major code
 	msg->data[14] = error_code & 0xFF; // Minor code
 	msg->data[15] = (error_code >> 8) & 0xFF; // Minor code
 
+	if (error_code == 0xE310) {
+		msg->data[3] = 0x2A;
+		msg->data[12] = 0x07;
```

Comment:
Better to still have those values documented.
```suggestion
		msg->data[3] = 0x2A; // General Information
		msg->data[12] = 0x07; // DIMM Error Type
```

Reply:
I have push a commit to add these comments.
Please help to review again, thanks.